tests/test-admin-locking.sh

```diff
@@ -25,10 +25,10 @@ set -euo pipefail
 setup_os_repository "archive-z2" "syslinux"
 
 # If parallel is not installed, skip the test
-if ! parallel --help >/dev/null 2>&1; then
-    echo "1..0 # SKIP no /usr/bin/parallel"
+if ! which parallel >/dev/null 2>&1; then
```

Comment:
Maybe `parallel --gnu /bin/true </dev/null 2>/dev/null` would be more correct; moreutils parallel could conceivably grow `--help` and `--version`, but hopefully nothing incompatible with GNU parallel is going to gain a `--gnu` option.